### ISSUE TITLE
you have big problems with remove command

### ISSUE THREAD

**Issue by qwertilus** [ParrotSec/tasksel#1]
this shit deleted 90% of my system with command `sudo tasksel remove lamp-server`

**Comment by PalinuroSec** [comment]
could you please provide more info to reproduce the issue?

**Comment by 0danteh** [comment]
@qwertilus I have submitted a pull request, hopefully it solves the problem. By the way, as a side note, you shouldn't perform `tasksel remove` on important groups of packages that are crucial for your PC's life.

**Comment by PalinuroSec** [comment]
does it still affect the system after 3.73+parrot5?

**Comment by 0danteh** [comment]
Yes, because he has basically deleted that group of packages that are vital to the system. The error, in my eyes, relies in the fact that the removal of certain commands needs to be properly checked before being executed. This is what I proposed in the pull requests @PalinuroSec.
